streaming/index.js

```diff
@@ -626,7 +626,7 @@ const startServer = async () => {
     const listener = message => {
       const { event, payload, queued_at } = message;
 
-      const transmit = () => {
+      const transmit = (payload) => {
```

Comment:
I would not re-use the same `payload` name as it is already defined in the above scope, this is confusing (and `eslint` should complain about it?)

Reply:
In my refactor on top of this branch I lifted the transmit method up in scope entirely, which resolves the shadowing of the payload variable here